src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionParsing.scala

```diff
@@ -113,34 +117,43 @@ trait SlickExpressionParser extends JavaTokenParsers {
   private def entityRootDot: Parser[RootFunc] =
     "this." ^^ { _ => entityRootFunc}
 
+  private val namespacedIdent: Parser[String] = rep1sep(ident, ":") ^^ {
```

Comment:
Could also possibly write this as a `Parser[AttributeName]` ?  Again, not sure of the implications